db/fixtures/lessons/git_lessons.rb

```diff
@@ -35,5 +35,15 @@ def git_lessons
       github_path: '/git/intermediate_git/using_git_in_the_real_world.md',
       identifier_uuid: 'c8b7ccc1-8a16-4545-9d46-b9091d45c6b4',
     },
+
+    },
```

Comment:
This is where the syntax error is coming from @CBID2 

```suggestion
    {
```